### ISSUE TITLE
bluetooth-audio doesn't work with new nvda

### ISSUE THREAD

**Issue by mpaulinelli** [mltony/nvda-bluetooth-audio#22]
Hi,
The application is not working at NVDA Version 2025. NVDA says Addon is compatible but it still doesn't work.
Thanks.
